styles.css

```diff
@@ -123,3 +126,250 @@ body { background: #e5e5e5; }
 
 .first { margin: 10px 10px; }
 .second { margin: 10px 0; }
+
+/**************
+  Works/about
+**************/
+
+.works-sect {
+  width: 375px;
+  height: 2788px;
+  left: 0;
+  top: 718px;
+  background: #e5e5e5;
+  background-color: #fff;
+  margin: 0 auto 0;
+}
+
+.project1 { margin-top: 114px; }
+
+.projects {
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-items: center;
+  background: rgb(243, 243, 243);
+}
```

Comment:
- [ ] As suggested by the previous reviewer, you should align your items using `flex` instead of grid. Please, replace it as it's required. You can follow this short [Youtbe Video](https://youtu.be/JJSoEo8JSnc) which can be helpful if you still struggling with how to implement it.

Reply:
[Kandy-Peter](https://github.com/Kandy-Peter) should i ignore this instruction then? 

In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

Reply:
Hello @TracK92 !

I've just realized that this point is not required in this project.
I am sorry to mislead you. I haven't done that on purpose :blush: